src/scripts/main.js

```diff
@@ -1,5 +1,28 @@
-'use strict';
+'use strict'; // Включаем строгий режим для безопасности
```

Comment:
Remove all comments on the page
```suggestion
'use strict';
```